content/doc/book/using/slave-master-access-control.adoc

```diff
@@ -0,0 +1,484 @@
+deployments have developed more sophisticated trust separation model,
+where the administrators of a master might take agents owned by other
+teams. In such an environment, agents are less trusted than the master.
+
+Starting 1.587 (and 1.580.1), Jenkins added a subsystem to put a wall
+between a master and an agent to safely allow less trusted agents to be
+connected to a master. Since Jenkins 2.0, this subsystem is enabled for
+all new installations of Jenkins. If you fit the "larger enterprise
+deployment" description above, then we highly recommend you turn this
+mode on. The following are examples of deployments that fit this
+profile:
+
+* You took agents that are managed by another person who is not Jenkins
+administrator because they have special requirement for their build jobs
+* You have some jobs that are configured to run on a specific agent
+because it is sensitive
+
+To turn this switch on, go to "Manage Jenkins" > "Configure Global
+Security", and check "Enable Agent → Master Access Control"
+
+On the other hand, if all your agents are trusted to the same degree as
+your master, then it is safe to leave this subsystem off. The following
+are examples of deployments that fit this profile:
+
+* You have set up your Jenkins master and all the agents by yourself
+* All the users of your Jenkins have the same access to all the jobs
+
+For this subsystem to function, it needs cooperation from plugins. And
+until plugins get updated to work in this new environment, this may
+cause seemingly legitimate operations to fail, resulting in broken
+builds and other failures.
+
+This Wiki page is intended to be the one-stop shop for those who are
+experiencing problems due to this subsystem.
+
+== My Jenkins is broken, help!
+
+If you see an exception like the following in console output, logs,
+etc., then you are affected by this access control mechanism:
+
+....
+java.lang.SecurityException: agent may not create /var/lib/jenkins/foo/bar/...
+See http://jenkins-ci.org/security-144 for more details
+
+java.lang.SecurityException: Sending org.jenkinsci.plugins.gitclient.CliGitAPIImpl$GetPrivateKeys from agent to master is prohibited.
+See http://jenkins-ci.org/security-144 for more details
+....
+
+The first message indicates that an agent tried to access a file on the
+master, and Jenkins has blocked that. It is possible that this indicates
+an active attack, but it is also possible that plugins that haven't yet
+upgraded to work with SECURITY-144 is doing something legitimate.
+
+The second message indicates that an agent has asked the master to
+execute a command, and Jenkins has blocked that because the command
+isn't classified by plugin developers whether it is safe to be executed
+from the agent.
+
+To resolve this problem, you have the following options:
+
+* https://wiki.jenkins.io/display/JENKINS//Slave+To+Master+Access+Control#SlaveToMasterAccessControl-update-plugins[Check
```

Comment:
This should be just a link to a section, e.g. `<<update-plugins>>`.

Reply:
@zbynek do you mind breaking down how to link to a section ? I'm thinking on line 76 `<<update-plugins>> [Check ..`

Reply:
@LarrySul please see https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#links , the part about *Internal cross references*